kale/embed/mpca.py

```diff
@@ -93,38 +129,57 @@ def fit(self, X):
         return self
 
     def transform(self, X):
-        """
-        Parameter:
-            X: array-like, shape (I1, I2, ..., n_samples)
-        ----------
-        Return:
-            Transformed data, array-like, shape (i1, i2, ..., n_samples)
+        """Perform dimension reduction on X
+
+        Args:
+            X (ndarray): shape (I1, I2, ..., n_samples)
```

Comment:
Is there a more specific name for X such as X_high_dim or X_original or X_orig

Reply:
Shall we use X_original for input (high-dimensional) data and X_transformed for output (low-dimensional) data

Reply:
Not necessarily, right? We can apply MPCA on transformed data as well. Maybe I am getting overcomplicated and keeping as simple as X is the best.